jsplot/src/test/java/tech/tablesaw/components/MarkerTest.java

```diff
@@ -16,4 +16,18 @@ public void asJavascript() {
     assertTrue(x.asJavascript().contains("symbol"));
     assertTrue(x.asJavascript().contains("size"));
   }
+
+  @Test
+  public void testCustomPalette() {
+    Marker palette = Marker.builder().colorScale(Marker.Palette.JET).build();
+    assertTrue(palette.asJavascript().contains("colorscale: 'Jet'"), palette.asJavascript());
+    Marker x = Marker.builder().addColorScale(0, 255, 0, 0).addColorScale(1, 0, 255, 0).build();
+    assertTrue(
+        x.asJavascript()
+            .contains(
+                "colorscale: [\n"
```

Comment:
this test seems a little brittle since it's sensitive to formatting

Reply:
I've had this PR in my inbox for many days since these minor remarks are, to me, a mountain of work.

I wonder if it's something obvious I'm not seeing about how to make this test less brittle (there are other existing tests sensitive to formatting) or how Jackson would simplify that little method?

The way I see it, to test the output in a non-brittle way would mean to basically *parse* that generated JS. Which isn't obvious how (Jackson, you say, but note we don't have JSON, we have Javascript there) and it would make the test much more complex.

As for introducing Jackson that's a good idea, but, again, nothing easy to hook into. I have not one but *two* branches where I try to see how Jackson might lock for the JS/JSON output.

This one https://github.com/emilianbold/tablesaw/commits/emi/jackson is from late last year and I was trying to use whole objects to serialize via Jackson. Not bad, but a lot of work.

This other one https://github.com/emilianbold/tablesaw/commits/jackson is from today and I'm using the easier way of writing the `Map<String,Object>` via Jackson. This seems better, but, of course, there's a lot of work to migrate everything to Jackson.

In conclusion, I don't know how to move this PR forward. I'll make a PR for the Jackson branch but for this one I don't know how to make it better and the code is starting to become stale. I guess I should just drop the PR.